### PR TITLE
Order questions by upvotes

### PR DESCRIPTION
**Pull request by RyanBillard**

### CODE REVIEW

File: AMA.iml [hunk 1]
```diff
@@ -8,7 +8,7 @@
       <sourceFolder url="file://$MODULE_DIR$/src/main/resources" type="java-resource" />
       <excludeFolder url="file://$MODULE_DIR$/target" />
     </content>
-    <orderEntry type="jdk" jdkName="1.8" jdkType="JavaSDK" />
+    <orderEntry type="inheritedJdk" />
```
Comment (SarrankanPathmanathan) [inline AMA.iml]: What does this do?

Comment (RyanBillard) [inline AMA.iml]: WHATEVER YOU WANT IT TO DO BABY


I don't actually know. I think it has something to do with how intelliJ locates the JDK

File: src/main/java/ama/post/CommentPost.java [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import javax.persistence.ManyToOne;
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Comparator;
```
Comment (stephanernst) [inline src/main/java/ama/post/CommentPost.java]: Not needed.

File: src/main/resources/templates/ama.html [hunk 3]
```diff
@@ -1,28 +1,41 @@
...
+                <div class="panel panel-default">
+                    <div class="panel-body">
+                <span th:text="|${comment.text} |"></span>
+                <span th:text="|(${comment.getPoints()}) |"/><a th:href="@{/comments/upvote(id=${comment.id})}">Upvote</a> <a th:href="@{/comments/downvote(id=${comment.id})}">Downvote</a>
+                <form th:action="@{/comments/__${comment.id}__/reply}" method="post">
+                    <input type="text" name="reply" /><input type="submit" value="Reply" />
+                </form>
+                    </div>
+                    <div class="panel-footer">
+                <div style="background-color: #fff;" class="well well-sm" th:each="reply : ${comment.getSortedReplies()}">
```
Comment (SarrankanPathmanathan) [inline src/main/resources/templates/ama.html]: need a better color

File: src/main/resources/templates/ama.html [hunk 4]
```diff
@@ -1,28 +1,41 @@
...
-</form>
-<h2>Comments</h2>
-<!--/*@thymesVar id="baseComments" type="java.lang.Iterable"*/-->
-<div th:each="comment : ${baseComments}">
-    <p><a th:text="${comment.text}">Title ...</a>
-    <a th:href="@{/comments/upvote(id=${comment.id})}">Upvote</a><span th:text="|(${comment.getUpvotes()})|"/> <a th:href="@{/comments/downvote(id=${comment.id})}">Downvote</a><span th:text="|(${comment.getDownvotes()})|"/>
-    </p>
-    <form th:action="@{/comments/__${comment.id}__/reply}" method="post">
-        <p><input type="text" name="reply" /><input type="submit" value="Reply" /></p>
+<div class="container-fluid">
```
Comment (AboudS777) [inline src/main/resources/templates/ama.html]: What does "fluid" mean for the container?

File: src/main/java/ama/post/PostController.java [hunk 5]
```diff
@@ -12,6 +12,7 @@
 import org.springframework.web.bind.annotation.*;
 
 import javax.servlet.http.HttpServletRequest;
+import java.util.Comparator;
```
Comment (stephanernst) [inline src/main/java/ama/post/PostController.java]: Not needed.

File: src/main/resources/templates/ama.html (hunk 4 above)
Comment (RyanBillard) [inline src/main/resources/templates/ama.html]: It means it will fill the entire available space as opposed to having a fixed width.

**Pull request merged**